src/lib/cfa/ntp_conf.rb

```diff
@@ -209,7 +210,9 @@ def reset_cache
       def record_entries
         return @record_entries if @record_entries
         matcher = Matcher.new do |k, _v|
-          RECORD_ENTRIES.include?(k.gsub("[]", ""))
```

Comment:
But that would include also not supported keys.. of course CFA will complain. Just thinking out loud..

Reply:
we now support any key that appear. I am not sure how to do it when we have bunch of implemented keys.

Reply:
Just expected only the misc commands for using the `MiscRecord` and maybe a `GenericRecord` for the rest not defined yet.